mitodl/mitxonline
# Adds transaction_type to the Transaction table

#### Pre-Flight checklist


#### What are the relevant tickets?
#467 

#### What's this PR do?
Fixes #467 

#### How should this be manually tested?
Run migrations - after this, the transaction_type field should be present in Transaction and all existing records should be set to "payment". 
After completing checkout, the new transaction record should be of type "payment".
